projects/observability/src/shared/components/explorer-filter-on-hover/explorer-filter-on-hover.component.ts

```diff
@@ -0,0 +1,41 @@
+import { ChangeDetectionStrategy, Component, Inject } from '@angular/core';
+import { IconType } from '@hypertrace/assets-library';
+import { NavigationService } from '@hypertrace/common';
+import { IconSize, POPOVER_DATA } from '@hypertrace/components';
+import { ExplorerService } from '../../../pages/explorer/explorer-service';
+import { ExplorerFilterOnHoverData } from './explorer-filter-on-hover.directive';
+
+@Component({
+  selector: 'ht-explorer-filter-on-hover',
+  styleUrls: ['./explorer-filter-on-hover.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `<div class="hover-filter-container">
+    <ht-icon
```

Comment:
I feel this should be wrapped inside an ht-link so that users can just open it in a new tab.